src/libertem/io/dataset/blo.py

```diff
@@ -211,6 +213,59 @@ def _read_header(self):
         with open(self._path, 'rb') as f:
             return np.fromfile(f, dtype=get_header_dtype_list(self._endianess), count=1)
 
+    def _read_metadata(self) -> tuple[str]:
+        """
+        Read metadata from the block after the inital 64 byte header
+        but before the first Data section (as interpreted from the header).
+
+        In the files alrady seen, this is a Windows-delimited sequence
+        of lines of text starting at byte 240.
+
+        As we aren't confident in the format this function doesn't try
+        to decode the metadata, it just loads it and cleans up any null
+        bytes or extra whitespace.
+        """
+        if self._header['MAGIC'] == 259:
+            metadata_start_pos = 240
+            data_start_byte = int(self._header['Data_offset_1'].item())
+            with open(self._path, 'rb') as f:
+                f.seek(metadata_start_pos)
+                metadata_bytes = f.read(data_start_byte - metadata_start_pos)
+            metadata = metadata_bytes.decode(errors='ignore').strip('\x00')
+            metadata = (m.strip() for m in metadata.splitlines())
+            return tuple(m for m in metadata if len(m) > 0)
+        return tuple()
+
+    @staticmethod
+    def _try_read_bitdepth(metadata: tuple[str]) -> str:
```

Comment:
```suggestion
    def _try_read_bitdepth(metadata: tuple[str, ...]) -> str:
```

`tuple[str]` is a tuple with a single element; for example:


```python
def f(t: tuple[str]):
    pass

f(("a",))
f(("a", "b", "c"))  # <- mypy: error: Argument 1 to "f" has incompatible type "tuple[str, str, str]"; expected "tuple[str]"  [arg-type]


def g(t: tuple[str, ...]):
    pass

g(("a",))
g(("a", "b", "c"))  # this is fine
```